pennylane/wires.py

```diff
@@ -63,7 +63,7 @@ def _process(wires):
         # of considering the elements of iterables as wire labels.
         wires = [wires]
 
-    if qml.math.get_interface(wires) == "jax" and not qml.math.is_abstract(wires):
+    if math.get_interface(wires) == "jax" and not math.is_abstract(wires):
```

Comment:
```suggestion
    if type(wires).__module__.split(".")[0] == "jax" and not math.is_abstract(wires):
```
Potentially even just doing this to get rid of the math dependency. Yes, it's as little more in-the-weeds than the nice `get_interface`, but might be worth it?

Reply:
But what about `math` to use `is_abstract`?

Reply:
Resolving, as we need is_abstract!